src/main/java/com/github/dockerjava/api/command/CreateContainerCmd.java

```diff
@@ -417,6 +417,8 @@
 
     CreateContainerCmd withTty(Boolean tty);
 
+    CreateContainerCmd withTmpFs(Map<String, String> tmpFs);
```

Comment:
I'm not sure to understand : you expect TmpFs to be only accessible with an HostConfig instance ? I mean, there's a lot of with* methods like this one, right ?